pkg/stats/stats_recorder.go

```diff
@@ -107,7 +119,10 @@ func (r *recorder) Stop() {
 }
 
 func (r *recorder) GetStats() Stats {
-	return <-r.getStatsChan
+	if stats, ok := <-r.getStatsChan; ok {
+		return stats
+	}
+	return r.latestStats.Stats()
```

Comment:
So this function will always return the last stats that were computed (if it's called after close)?